### PR TITLE
Makes hulk more susceptible to cold damage

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
tweak: Hulks now get more damage from freeze damage, take it in brute, and get the damage at higher temps than normal
/:cl:

requested by oranges, makes the temp-gun better against hulkies

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1816,15 +1816,21 @@ GLOBAL_LIST_EMPTY(roundstart_races)
...
 
-	// Apply some burn damage to the body
-	if(humi.coretemperature < bodytemp_cold_damage_limit && !HAS_TRAIT(humi, TRAIT_RESISTCOLD))
+	// Apply some burn / brute damage to the body (Dependent if the person is hulk or not)
+	var/is_hulk = HAS_TRAIT(humi, TRAIT_HULK)
+
+	var/cold_damage_limit = bodytemp_cold_damage_limit + (is_hulk ? BODYTEMP_HULK_COLD_DAMAGE_LIMIT_MODIFIER : 0)
+
+	if(humi.coretemperature < cold_damage_limit && !HAS_TRAIT(humi, TRAIT_RESISTCOLD))
+		var/damage_type = is_hulk ? BRUTE : BURN
```
Comment (EgorDinamit) [inline code/modules/mob/living/carbon/human/species.dm:1825]: Why should it deal brute damage to Hulks?

Comment (Qustinnus) [reply code/modules/mob/living/carbon/human/species.dm:1825]: dunno oranges asked me for this

**Comment by EgorDinamit** [comment]
Personally, I'd like hulk nerfs, but people still can get Hulk + Space Adaptation to avoid cold damage.

**Comment by Tlaltecuhtli** [comment]
make hulk weak to pepper spray

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/mob/living/carbon/human/species.dm:1825]: theres icicles forming in your body that cause bodily damage

**Comment by Qustinnus** [comment]
blessed

**Comment by ATH1909** [comment]
This isn't how you make the temp gun better, IMO. Hulks are already weak to mutadone; making them weak to cold-based weapons as well (which are about as hard to get as mutadone, especially on short notice as a hulk is breaking into your department) is just kind of redundant.

I think the better way to do this would be to make the temperature gun a more viable weapon against everyone (who isn't in a hardsuit), not just hulks. Maybe make it disable natural body temperature adjustment for a period of time after each hit? That way, everyone would be about as weak to it as lizards are currently (for as long as the status effect lasts, anyway).

**Comment by ATH1909** [comment]
Also, forgive me for my ignorance, but do you have any justification as to _why_ hulks should logically be weak to cold? Is that an established weakness of the Hulk in the comics?

**Comment by Qustinnus** [comment]
bro do we look like marvel? we dont need to adhere to marvel lore

**Comment by ATH1909** [comment]
> bro do we look like marvel?

But the mutation is a blatant reference to marvel.

> we dont need to adhere to marvel lore

True, but how is someone going to make the logical leap that "hey, this guy looks like a hulk, maybe I should use cold weaponry on him!" without memorizing the finer details of the mutations page of the wiki/codediving?

**Comment by ATH1909** [comment]
Do hulks even care about the slowdown from being cold currently? If we just buffed the temp gun to be not hot garbage against everyone, then they'd be a hulk soft counter (which can in turn be countered by space adaptation, space suits, etc., just like the change from this PR is) by default, as they'd let you kite them pretty easily (like a bola does) by chilling them enough to invoke the cold movement speed penalty(/penalties).

**Comment by Qustinnus** [comment]
no im making space adaption not work when hulk is on

**Comment by ATH1909** [comment]
> no im making space adaption not work when hulk is on

how does that solve any of the problems with hulk?

**Comment by necromanceranne** [comment]
To be honest, the fact that hulks get slowed by cold but not from being riddled with bullets is pretty weird. I actually agree that this is such a departure from the original concept that at a certain point I think we might want to distance the mutation entirely from Marvel's Hulk.

Intellectually Distinct Green Bulked Fellow.

**Comment by Qustinnus** [comment]
cant be immune to freeze

**Comment by ATH1909** [comment]
> To be honest, the fact that hulks get slowed by cold but not from being riddled with bullets is pretty weird. I actually agree that this is such a departure from the original concept that at a certain point I think we might want to distance the mutation entirely from Marvel's Hulk.
> 
> Intellectually Distinct Green Bulked Fellow.

slowdown from bullets is due to pain, which just makes hulks madder. slowdown from cold is from your body slowing down and stuff, I think.

**Comment by Qustinnus** [comment]
if you dont watch out I'll make it so people can sing the hulk out of his mutation

**Comment by optimumtact** [comment]
lol, someone is mad

**Comment by Tlaltecuhtli** [comment]
well if i see hulk griefing i m gonna go grab laser from armory, with this in...... there is another round start laser-like gun that only works on hulks........ kinda weak.............

**Comment by ATH1909** [comment]
> lol, someone is mad

Yes, because this change doesn't accomplish anything other than annoying hulks who have to walk through hull breaches. I'm saying this as a nonhuman main, mind you.

**Comment by optimumtact** [comment]
> well if i see hulk griefing i m gonna go grab laser from armory, with this in...... there is another round start laser-like gun that only works on hulks........ kinda weak.............

good for you?

**Comment by optimumtact** [comment]
> > lol, someone is mad
> 
> Yes, because this change doesn't accomplish anything other than annoying hulks who have to walk through hull breaches. I'm saying this as a nonhuman main, mind you.

It makes the tempgun have a use

**Comment by ATH1909** [comment]
> > > lol, someone is mad
> > 
> > 
> > Yes, because this change doesn't accomplish anything other than annoying hulks who have to walk through hull breaches. I'm saying this as a nonhuman main, mind you.
> 
> It makes the tempgun have a use

Not really, because seccies will still just use lasers and mutadone and stuff instead, because the tempgun is garbage, even with doubled damage.

Also, there are better ways to make tempguns have a use/be hulk counters than this PR, which I outlined above.

**Comment by Jared-Fogle** [comment]
hey guys, i've read every marvel comic and this is one of his weaknesses. here'sproof

image

**Comment by optimumtact** [comment]
I disagree

**Comment by ATH1909** [comment]
> hey guys, i've read every marvel comic and this is one of his weaknesses. here'sproof

that's just an image of the hulk in a cold place.

**Comment by optimumtact** [comment]
> > hey guys, i've read every marvel comic and this is one of his weaknesses. here'sproof
> 
> that's just an image of the hulk in a cold place being mildly annoyed by some cold, not being damaged/having his life threatened by it.

He actually dies of sadness, not the cold

**Comment by ATH1909** [comment]
like, if it was an established weakness that's been brought up multiple times and been shown to be more effective than normal, like the flash's weakness to cold, then I could be on board with this.

but it looks like the hulk is sitting near-naked on an alien planet there that's probably cold enough to kill a normal human, and he's just mildly annoyed by the cold. if anything, that image makes the hulk look RESISTANT to cold, just not AS resistant to cold (in a non-raging scenario) as he is to, say, bullets (while raging).

**Comment by necromanceranne** [comment]
> cant be immune to freeze

So spacesuits don't work.

**Comment by Tlaltecuhtli** [comment]
make hulk weak to radiations and give armory 5 bars of uranium like for super man

**Comment by Qustinnus** [comment]
>give sec 5 bars of uranium

lol

**Review by Mothblocks — APPROVED** [review]

**Comment by Tlaltecuhtli** [comment]
uranum
here is image of hulk from superheroes comics weak to uranium for lore reasonings

**Comment by D4C-420** [comment]
@optimumtact Out of curiosity why does getting cold do brute damage to hulks. I mean is there a gameplay reason I'm missing as for why or is it a lore reason why? With the way damage from temp works (burn damage) it does not make much sense.  Also if you want the temp gun to do shit/ people to use it nerfing hulks is not a good way to do it cause there is already a general (lasers/bolas) and niche (mudaone) way to deal with hulks.

**Comment by optimumtact** [comment]
> @optimumtact Out of curiosity why does getting cold do brute damage to hulks. I mean is there a gameplay reason I'm missing as for why or is it a lore reason why? With the way damage from temp works (burn damage) it does not make much sense.

The big veins in their body means their blood can freeze into crystals and kill them.

>Also if you want the temp gun to do shit/ people to use it nerfing hulks is not a good way to do it cause there is already a general (lasers/bolas) and niche (mudaone) way to deal with hulks.

I disagree, I think it will make the tempgun have more use.

**Comment by D4C-420** [comment]
@optimumtact sure lets roll with that. The way you word it makes it seem like they should choke to death but maybe its pierces their flesh. Still combined with #56132 making temp guns bulky combined with #56133 limiting how easy it is to mass produce tempgun's along with how easy it is to warm yourself up with a little preparation (hot coffee, anti-freeze) this is just a hulk nerf not a tempgun buff. Want to buff the tempgun? Make it more effective against armored foes and those protected against temperature.

**Comment by Cartographer-D** [comment]
> 
> 
> @optimumtact sure lets roll with that. The way you word it makes it seem like they should choke to death but maybe its pierces their flesh. Still combined with #56132 making temp guns bulky combined with #56133 limiting how easy it is to mass produce tempgun's along with how easy it is to warm yourself up with a little preparation (hot coffee, anti-freeze) this is just a hulk nerf not a tempgun buff. Want to buff the tempgun? Make it more effective against armored foes and those protected against temperature.

Don't drag me into this!

Also the temp gun is really good against changelings, just FYI.

**Comment by CRITAWAKETS** [comment]
hulk nerf idea number 126216: make high mood get rid of hulk status

**Comment by ATH1909** [comment]
> hulk nerf idea number 126216: make high mood get rid of hulk status

or make the strength of hulk punches be dependent on your mood and how much damage you have

**Comment by D4C-420** [comment]
> or make the strength of hulk punches be dependent on your mood and how much damage you have

Time to pwr.gm my way to depression with hulk and smash da noobs.

**Comment by Jared-Fogle** [comment]
im going to make the distro cold when theres a hulk, this is a good change

**Comment by ExcessiveUseOfCobblestone** [comment]
Can we go ahead and start calling hulking a disability for genetics I don’t think it’s that good.

If you want the temp gun to see use have it set your average body temp lower for a small duration (stackable), the reason it’s Garbo is because you have to keep shooting it to maintain the temp unless you have like a space suit on where it does it for you

**Comment by optimumtact** [comment]
You're my favourite disability in genetics cobby

**Pull request merged**